create_datasets.py

```diff
@@ -0,0 +1,40 @@
+"""Script to create Sentiment140 datasets."""
+
+import argparse
+
+import numpy as np
+import pandas as pd
+
+import utils
+
+parser = argparse.ArgumentParser(
```

Comment:
The parser is only necessary when runing the script directly therefore, I would put it inside the `if __name__ == '__main__'`condition

Reply:
Corrected in https://github.com/XanX3601/deep_sentiment140/pull/3/commits/4833fdfb78d9b6d5d7314cc5c237879d8ad4c360.